build/dashboard.css

```diff
@@ -1,34 +0,0 @@
-.dashboard-page-container {
```

Comment:
If these .css files are _actually_ built files (in that they are transpiled from .scss files) then they shouldn't be tracked in Git. Instead, only the source files should be tracked and the CSS files should be built locally during dev and built via CI during deployment. Otherwise, this creates a scenario where the built files can be out of sync with the source files. Imagine if you changed a source file, but forgot to commit the built files? This would lead to a confusing scenario where you _swore_ you updated the source files but the changes weren't reflected in the web app.

Reply:
So I should just not include the built css files in the commit?

Reply:
Correct, you should not track generated files like this or transpiled JS. In fact, the directory where these built files go should be added to the `.gitignore` file. There are several reasons why you _shouldn't_ track generated files:

1. The aforementioned out-of-sync issue.
2. Adds the unnecessary task of having to commit changes to both the source and distribution files.
3. If your relying on building and tracking files locally, it's possible the built version of the app was built with outdated dependencies (for instance, if someone updated packages but you didn't downloaded the latest) which would be caught if the build was happening in CI.

If you look at the [Apprenticeship Project Starter](https://github.com/sparkbox/apprenticeship-project-starter) you can see that we're specifically disabling tracking the of the [`dist`](https://github.com/sparkbox/apprenticeship-project-starter/blob/main/.gitignore#L4).